introduction/modules/modules.md

```diff
@@ -105,6 +105,11 @@ The milestones module offers a framework to keep track of how a process is progr
 in the admin interface, and can be linked to specific tasks of a process. When that task is completed,
 the corresponding milestone is reached.
 
+### Notes
```

Comment:
Maybe rename this to 'Case notes'?

Reply:
The name of the module is 'Notes'

Reply:
Ah, I didn't check the file this was written in.